russian/sweet-bread/sweet-bread.md

```diff
@@ -0,0 +1 @@
+Внезапно раздался звонок в дверь. Открыв, вы увидели человека с тарелкой в руках, он мямлил что-то вроде "Просто покушать принёс тебе… ты бы покушал хоть, когда усталый!.. Покушать принёс чуть-чуть еды, это сладкий хлеб!.."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.